ocaml/idl/markdown_backend.ml

```diff
@@ -130,7 +130,7 @@ let markdown_section_of_message printer x =
   printer (sprintf "%s (%s)"
     x.msg_name
     (String.concat ", "
-      ((if x.msg_session then "session_id s" else "")::
+      ((if x.msg_session then ["session ref session_id"] else []) @
```

Comment:
I'd rather we kept the param names. Consider as a rough example `add_to_other_config(blah ref, string, string)`. I appreciate that the arguments are listed in order underneath, but it's much more obvious with a single look that it's actually `add_to_other_config(blah, key. value)` and not `add_to_other_config(blah, value. key)`.

Had I been around and asked about the online docs, I would have said no too 😄 . Actually, those are even less user friendly as it requires a click to expand and see the param names.